src/ASM/FiniteElement.h

```diff
@@ -117,19 +117,19 @@ class MxFiniteElement : public FiniteElement
   explicit MxFiniteElement(const std::vector<size_t>& n, size_t ip = 0);
 
   //! \brief Returns the number of bases.
-  virtual size_t getNoBasis() const { return 1+M.size(); }
+  size_t getNoBasis() const override { return 1+M.size(); }
```

Comment:
So the analyzer also wants you to use the `override` keyword as well? That's fine, but there are lots of other classes where virtual methods have been overridden without the keyword, in addition to those changed here. So why these and not the others? Or will the analyzer complain on others only when these are fixed, or what. I'm only trying to understand the logics here since the commit log does not give many hints ;)

Reply:
Mostly a 'I was in these files already so let's try to sneak in a few of them while at it'. And yes, cppcheck complains about all of these.